src/Stratis.Bitcoin/Utilities/NodeStats.cs

```diff
@@ -62,7 +66,7 @@ public string GetStats()
             lock (this.locker)
             {
                 string date = this.dateTimeProvider.GetUtcNow().ToString(CultureInfo.InvariantCulture);
-                statsBuilder.AppendLine($"======Node stats====== {date}");
+                statsBuilder.AppendLine($"======Node stats====== {date} ====== IBD:({this.NodeIsInIbd})");
```

Comment:
IBD status probably should be on a separated line after the `====Node stats` header